src/components/dashboard/dashboard-list/DashboardListButtons.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+
+interface IDashboardListButtonsProps {
+    mode: string;
+}
+
+class DashboardListButtons extends React.Component<IDashboardListButtonsProps> {
+    setSliders = (): React.ReactNode => {
+        switch (this.props.mode) {
+            case 'Application Reviewed':
+                return (
+                    <div className="level-left">
+                        <div className="level-item">
+                            <p>Application Score </p>
+                            <input
+                                className="slider has-output-tooltip is-fullwidth is-large"
+                                step="0.5"
+                                min="0"
+                                max="5"
+                                value="3"
+                                type="range"
+                            ></input>
+                            <output htmlFor="sliderWithValue">3</output>
+                        </div>
+                    </div>
```

Comment:
It looks like this block is repeated for all of the cases, when this occurs its better to separate this block into a single component that is referenced. If there are small changes that are required, then I would probably suggest to pass as prop. What are the differences between this and the rest of the cases? I see that this one has `is-fullwidth` but that's it.

Reply:
Ohh I see now, should be fixed! Thanks